src/hooks/useQuery.ts

```diff
@@ -3,15 +3,15 @@ import { useMemo } from 'react'
 import { sanitizeInput, sanitizeNegativeAndMakeMultipleOf } from 'utils'
 import { DEFAULT_FORM_STATE } from 'components/TradeWidget'
 
-export function useQuery(): { sellAmount: string; buyAmount: string; validUntil?: string } {
+export function useQuery(): { sellAmount: string; price: string; validUntil?: string } {
```

Comment:
Did you update the default route on App.tsx as well?

Reply:
Now i do. Thanks! b3528e04376687c98fed3c10d35a4a77ba908e49